ApptentiveCore/ApptentiveCore/Transforms/Metadata.xml

```diff
@@ -0,0 +1,16 @@
+﻿<metadata>
+<enum name="apptentive.com.android.util.LogLevel, ApptentiveCore">
+<field name="Verbose" javaname="VERBOSE" />
+<field name="Debug" javaname="DEBUG" />
+<field name="Info" javaname="INFO" />
+<field name="Warning" javaname="WARNING" />
+<field name="Error" javaname="ERROR" />
+</enum>
+  
```

Comment:
Why these nodes are removed?

Reply:
Its not removed just normalize the naming.

Reply:
I meant the <remove-node> here. Just didn't understand what does it do here.